packages/cache/src/LocalCacheStorage.ts

```diff
@@ -14,45 +13,28 @@ export class LocalCacheStorage extends CacheStorage {
     return path.join(this.internalCacheFolder, hash);
   }
 
-  protected async _fetch(
-    hash: string,
-    outputFolder: string | string[]
-  ): Promise<boolean> {
+  protected async _fetch(hash: string): Promise<boolean> {
     const localCacheFolder = this.getLocalCacheFolder(hash);
 
     if (!fs.pathExistsSync(localCacheFolder)) {
       return false;
     }
 
-    await Promise.all(
-      outputFolderAsArray(outputFolder).map(async folder => {
-        const locationInCache = path.join(localCacheFolder, folder);
-
-        if (!fs.pathExistsSync(locationInCache)) {
-          return;
-        }
-
-        await fs.mkdirp(folder);
-        await fs.copy(locationInCache, folder);
-      })
-    );
+    fs.readdirSync(localCacheFolder).forEach(fileOrFolder => {
+      fs.copySync(path.join(localCacheFolder, fileOrFolder), fileOrFolder);
+    });
 
     return true;
   }
 
-  protected async _put(
-    hash: string,
-    outputFolder: string | string[]
-  ): Promise<void> {
+  protected async _put(hash: string, outputGlob: string[]): Promise<void> {
     const localCacheFolder = this.getLocalCacheFolder(hash);
 
-    await Promise.all(
-      outputFolderAsArray(outputFolder).map(async folder => {
-        const outputFolderInCache = path.join(localCacheFolder, folder);
-
-        await fs.mkdirp(outputFolderInCache);
-        await fs.copy(folder, outputFolderInCache);
-      })
-    );
+    const files = fg.sync(outputGlob);
+    files.forEach(file => {
```

Comment:
Same question as above, about whether `file` will be absolute or relative here.

Reply:
it is relative